src/main/java/mate/academy/spring/validator/EmailValidator.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]"
+            + "+\\\\.[A-Za-z]{2,}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        final Pattern patterns = Pattern.compile(EMAIL_PATTERN);
```

Comment:
move this final variable up to class field static constants, and fix name without using plural form.

Reply:
got it)